### ISSUE TITLE
Regressions from 4.4.0 to 4.5.0

### ISSUE THREAD

**Issue by dstj** [mysticmind/reversemarkdown-net#393]
Just tried upgrading to v4.5.0 from v4.4.0 and my unit test fails because a simple conversion is now different:
* 1 less space after "regular text", replace by an extra blank line before the `<p>`.
* Different line break sequence in the `<ul><li>` list. This is strange.

See the following code for a reproduction:

```csharp
const string html = "This is regular text\n<p class=\"c1\">This is HTML: <ul><li>Line 1</li><li>Line 2</li><li><mark>Line 3 has an unknown tag</mark></li></ul></p>";
var converter = new Converter(new Config { UnknownTags = Config.UnknownTagsOption.Bypass, ListBulletChar = '*' });
var markdown = converter.Convert(html).Trim();
const string markdownConversion = "This is regular text \r\nThis is HTML: \r\n\r\n* Line 1\r\n* Line 2\r\n* Line 3 has an unknown tag";
// with v4.5.0 const string markdownConversion = "This is regular text\r\n\r\nThis is HTML: \r\n\r\n* Line 1\n* Line 2\n* Line 3 has an unknown tag";
markdown.Should().Be(markdownConversion);
```

The [changelog for 4.5.0](https://github.com/mysticmind/reversemarkdown-net/compare/4.4.0...4.5.0) does not mention anything that seems related.

Are these regressions or expected modifications? Thanks.

**Comment by mysticmind** [comment]
Thanks for reporting the issue, will have a look and revert to you.

**Comment by mysticmind** [comment]
Quick follow on note, the different line break sequence looks like a regression. Rest are all expected changes.

**Comment by mysticmind** [comment]
Fix available in v4.6.0
